.github/CODEOWNERS

```diff
@@ -10,10 +10,8 @@
 core/*                    @dhermes
 test_utils/*              @dhermes
 
-# Jon Wayne Parrott is the author of the new google.api.core library.
-# NOTE: This trumps the previous core/* directive, since the last match is
-#       what matters.
-core/google/api/core/*    @jonparrott
+# Jon Wayne Parrott is the author of the new google.api_core library.
```

Comment:
Do we want to call it "new" in here? I assume this doc will exist long after it isn't new.

Reply:
I expect this doc will cease to exist when we split the repo.

Reply:
DERP